main.py

```diff
@@ -831,6 +824,87 @@ def double_different(
                 phaseB, hB, pB = PhaseIdx.single_exc(getattr(det_i, spin_b), getattr(det_j, spin_b))
                 yield (a, b), phaseA * phaseB
 
+    @staticmethod
+    def double_same_unique_external(idx, psi_i, psi_j, spindet_a_occ_i, spindet_a_occ_j, exc, spin):
```

Comment:
Please remove the `_internal` and commit only the `external` by removing the name.

I would like to keep this code with the only "required" optimization